src/components/SocialShareButtons.vue

```diff
@@ -51,9 +40,9 @@ export default {
   $social-button-transition: all 0.5s ease;
   $social-button-margin: 0.25rem;
 
-  $social-brand-facebook: #3b5998;
-  $social-brand-twitter: #55acee;
-  $social-brand-pinterest: #c32aa3;
+  $social-brand-facebook: #4267b2;
+  $social-brand-twitter: #1da1f2;
+  $social-brand-pinterest: #e60023;
```

Comment:
Any particular reason for the color changes?

Reply:
The color of Pinterest was very pink-like and did not match the color on the standard 'Pin it' button. So I investigated and found that none of the buttons were the original colors of the brands they represented so I updated them to the official branding colors.